server/players.py

```diff
@@ -35,18 +35,21 @@ def __init__(
         permission_group: int = 0,
         lobby_connection: "LobbyConnection" = None
     ):
-        super().__init__(player_id, login)
+        self._faction = 0
+        self._global_rating = (1500, 500)
+        self._ladder_rating = (1500, 500)
+
+        self.id = player_id
+        self.login = login
 
         # The player_id of the user in the `login` table of the database.
         self.session = session
         self.ip = ip
 
-        if global_rating is None:
-            global_rating = (1500, 500)
-        if ladder_rating is None:
-            ladder_rating = (1500, 500)
-        self.global_rating = global_rating
-        self.ladder_rating = ladder_rating
+        if global_rating is not None:
+            self.global_rating = global_rating
+        if ladder_rating is not None:
+            self.ladder_rating = ladder_rating
```

Comment:
Bad naming for `global_rating`, `self.global_rating` and `self._global_rating`, how about to create and use `set_global_rating()`?

Reply:
This commit does not change existent naming, and the `name` property backed by a `_name` attribute is pretty common practice. The property already acts like a setter and has no side effects beyond setting the backing value anyway.